guides/common/modules/proc_providing-feedback-on-red-hat-docs.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+
+[preface]
+[id="proc_providing-feedback-on-red-hat-documentation_{context}"]
```

Comment:
`missing attribute: context`

Reply:
The Configuring_Load_Balancer guide doesn't define a `context` attribute in its `master.adoc` and thus your inclusion fails as it relies on the `context` attribute being present.

The same is true for the RN doc, but I think you didn't edit that one?

```console
% find . -name master.adoc | xargs grep --files-without-match ':context:'
./guides/doc-Release_Notes/master.adoc
./guides/doc-Configuring_Load_Balancer/master.adoc
```

Reply:
Hi @evgeni , thank you so much for the guidance!  Got it. So I've added that context to master.adoc for the Load balancing guide in my last commit. You're right, I haven't touched the master.adoc for the RNs (Satellite has RNs written in the downstream repo).
Thank you!